great_expectations/cli/cli.py

```diff
@@ -471,6 +472,91 @@ def profile(datasource_name, data_assets, profile_all_data_assets, directory, vi
         )
 
 
+@cli.command()
+@click.argument("data_asset_name")
+@click.argument("suite_name")
+@click.option(
+    '--batch-kwargs',
+    default=None,
+    help='Additional keyword arguments to be provided to get_batch when loading \
+    the data asset. Must be a valid JSON dictionary'
+)
+@click.option(
+    "--directory",
+    "-d",
+    default=None,
+    help="The project's great_expectations directory.",
+)
+@click.option(
+    "--jupyter-notebook",
+    "-jn",
+    default=True,
+    is_flag=True,
+    help="By default open in browser unless you specify the --no-jupyter flag",
+)
+@click.option(
+    "--jupyter-lab", "-jl", is_flag=True, default=False, help="Edit in jupyter lab",
+)
```

Comment:
This seems slightly confusing, since you could specify inconsistent options and not get a tailored error (e.g. `--jupyter-notebook --jupyter-lab --no-jupyter`)

Reply:
Agreed.